tests/wpt/metadata/html/browsers/history/the-history-interface/001.html.ini

```diff
@@ -92,11 +40,16 @@
     expected: FAIL
 
   [state data should be a clone of the original object, not a reference to it]
-    expected: FAIL
+    expected:
+      if debug and (os == "mac") and (version == "OS X 10.12") and (processor == "x86_64") and (bits == 64) and (backend == "webrender"): PASS
```

Comment:
OK, that is weird. Normally I'd say ask on irc, but it's a bit of a ghost town atm.